commit 512d8c64431507208a5c9ec259fd772b8dfd27bc (refs/changes/41/5241/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-12-14T09:50:11-08:00 (10 months ago)
    
    TRITON-1034 Provisioning a VM with `do_not_inventory` set causes the workflow job to time out at putVm

diff --git a/lib/workflows/job-common.js b/lib/workflows/job-common.js
index 0a91129..5b9f560 100644
--- a/lib/workflows/job-common.js
+++ b/lib/workflows/job-common.js
@@ -304,6 +304,12 @@ function pollTask(job, cb) {
 function putVm(job, cb) {
     var vmapi;
 
+    if (job.params.payload.do_not_inventory) {
+        cb(null,
+            'VM has do_not_inventory set, no need to putVm to VMAPI');
+        return;
+    }
+
     /*
      * Checks (polls) the state of a machine in VMAPI. It is used for provisions
      * and VM actions such as reboot and shutdown.
diff --git a/package.json b/package.json
index 95e3989..a4df7f3 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.6.5",
+  "version": "9.6.6",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
